msdk-features/msdk-row-filtering/pom.xml

```diff
@@ -0,0 +1,43 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.github.msdk</groupId>
+        <artifactId>msdk-features</artifactId>
+        <version>0.0.2-SNAPSHOT</version>
+    </parent>
+    <artifactId>msdk-row-filtering</artifactId>

```

Comment:
Thomas, let's try to maintain the module naming notation according to the msdk directory structure. This module should have artifactId msdk-features-row-filtering